recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   - bob.extension
   - bob.blitz
   - bob.core
-  - hdf5
+  - hdf5 1.8.15*
```

Comment:
> This package would be OK with hdf5 1.8.16

OK, that is good to know. I noticed that continuum pin to the patch version, but sounds like you think we can patch to the minor version? Out of interest, how do you know?